src/components/reusable/modal/Modal.tsx

```diff
@@ -27,6 +28,7 @@ export default function Modal({
   isActive,
   footerChildren,
   backgroundColor,
+  increaseHeight = false,
```

Comment:
i think you don't need default value here, if increaseHeight passed it will have passed value otherwise it can/will be converted to false